web/html/src/components/input/Form.js

```diff
@@ -84,122 +78,56 @@ class Form extends React.Component<Props, State> {
     if (this.props.onValidate) {
       this.props.onValidate(allValid);
     }
-  }
-
-  validate(component: React.ElementRef<any>, props: Object) {
-    const { name } = props;
-    const results = [];
-    let isValid = true;
-
-    if (!props.disabled && (this.state.model[name] || props.required)) {
-      if (props.required && !this.state.model[name]) {
-        isValid = false;
-      } else if (props.validators) {
-        const validators = Array.isArray(props.validators) ? props.validators : [props.validators];
-        validators.forEach((v) => {
-          results.push(Promise.resolve(v(`${this.state.model[name] || ''}`)));
-        });
-      }
-    }
-
-    Promise.all(results).then((result) => {
-      result.forEach((r) => {
-        isValid = isValid && r;
-      });
```

Comment:
This part is missing from the new `validate()` function. This is where all the validator results get collected so it's critical.